task/github-add-comment/0.6/README.md

```diff
@@ -0,0 +1,152 @@
+# Add a comment to an issue or a pull request
+
+The `github-add-comment` task let you add a comment to a pull request or an
+issue.
+
+## Changelog
+
+- Added support for replacing a comment. It works by adding an invisible tag into the comment, which allows for
```

Comment:
This changelog is not valid I guess for this

Reply:
Good point, it's not. Should we even have a changelog? I'd say you can 'easily' find this info with git